indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -168,6 +168,16 @@
         has-variables (or (:has-variables variable-docs) (:has-variables service-docs))]
     (merge variable-docs service-docs {:has-variables has-variables})))
 
+(defn- get-granule-data-format
+  "Returns the Format field from
+  ArchiveAndDistributionInformation -> FileDistributionInformation"
+  [file-distribution-information]
+  (->> file-distribution-information
+       (map #(:Format %))
+       distinct
+       (remove #(nil? %))
```

Comment:
ultra-nit: `(remove #(nil? %))` can just be `(remove nil?)`. Same with `(map :Format)`

Reply:
Ohhhh yeah that's right! I couldn't remember the nicer way to do this so thanks